qt/scientific_interfaces/Inelastic/Manipulation/InelasticDataManipulationElwinTabModel.cpp

```diff
@@ -117,6 +126,45 @@ std::string InelasticDataManipulationElwinTabModel::createGroupedWorkspaces(Matr
   AnalysisDataService::Instance().remove("specWSnext");
   return workspace->getName() + "_extracted_spectra";
 }
+void InelasticDataManipulationElwinTabModel::setOutputWorkspaceNames(std::string const &workspaceBaseName) {
+  auto elwinSuffix = "_elwin_";
+  m_outputWorkspaceNames["qWorkspace"] = workspaceBaseName + elwinSuffix + "eq";
+  m_outputWorkspaceNames["qSquaredWorkspace"] = workspaceBaseName + elwinSuffix + "eq2";
+  m_outputWorkspaceNames["elfWorkspace"] = workspaceBaseName + elwinSuffix + "elf";
+  m_outputWorkspaceNames["eltWorkspace"] = workspaceBaseName + elwinSuffix + "elt";
+}
+
+std::string InelasticDataManipulationElwinTabModel::getOutputWorkspaceNames() const {
+  std::vector<std::string> keys = {"qWorkspace", "qSquaredWorkspace", "elfWorkspace", "eltWorkspace"};
+  std::ostringstream oss;
+  std::transform(keys.cbegin(), keys.cend(), std::ostream_iterator<std::string>(oss, ","),
+                 [&outNames = m_outputWorkspaceNames](const auto &key) { return outNames.at(key); });
```

Comment:
I think it may be problematic, since `m_outputWorkspaceNames` is an unordered map. That's why I have to make the last change (test were failing as the function could return the string in a different order than the one intended in the vector of keys)

Reply:
Ah ok, thanks for explaining!